### ISSUE TITLE
Can't Open Notes With Blank Name

### ISSUE THREAD

**Issue by bendangelo** [nvbn/everpad#78]
Created a note with an empty name. Now I can't open it. Sometimes it crashes as well.

**Comment by fmoo** [comment]
Are you running trunk or the latest release?

**Comment by bendangelo** [comment]
Not sure I installed it with this:

```
sudo add-apt-repository ppa:nvbn-rm/ppa
sudo apt-get update && sudo apt-get install everpad
```

**Comment by fmoo** [comment]
I _think_ this got fixed with #76.  @nvbn hasn't built a new release with it yet, though

**Comment by bendangelo** [comment]
Thanks
